ironfish/src/blockchain/blockchain.ts

```diff
@@ -175,6 +175,12 @@ export class Blockchain {
     this.autoSeed = options.autoSeed ?? true
     this.consensus = new TestnetParameters()
 
+    // TODO: Remove this during network reset
+    this.invalid.set(
```

Comment:
Since `invalid` is an LRU, do we need to worry about this getting evicted?

Reply:
it could, eventually, yeah.